code/modules/jobs/job_types/security_officer.dm

```diff
@@ -281,6 +281,14 @@ GLOBAL_LIST_INIT(available_depts, list(SEC_DEPT_ENGINEERING, SEC_DEPT_MEDICAL, S
 	id = /obj/item/card/id/syndicate_command/crew_id
 	backpack_contents = list(/obj/item/storage/box/survival/syndie=1)
 
+/datum/outfit/job/security/pirate/pilot
+	name = "First mate(pilot)"
```

Comment:
make this normal

Reply:
what do you mean with normal?

Reply:
it does not follow the other outfits in "grammar"